plugin/audit_log/tests/mtr/audit_log_default_db.test

```diff
@@ -47,7 +49,7 @@ connection test;
 SELECT * FROM t;
 use 	`db1`;
 --error ER_DBACCESS_DENIED_ERROR
-change_user user2,111,db1;
+change_user user2,111,db1,false;
```

Comment:
yeah... it was my first intention to name them this way. Now I did

http://jenkins.percona.com/view/5.7/job/mysql-5.7-param/399/